Document Matter integration requirements and support

djansen1987 opened the pull request:
Added information about setting up Matter integration with Thread and Home Assistant Connect ZBT-1 support.

<!--
  You are amazing! Thanks for contributing to our project!
  Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).

  Before submitting your pull request, please verify that you have chosen the correct target branch,
  and that the PR preview looks fine and does not include unrelated changes.
-->
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->



## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

djansen1987 committed:
Document Matter integration requirements and support

Added information about setting up Matter integration with Thread and Home Assistant Connect ZBT-1 support.

djansen1987 commented on source/_integrations/nuki_matter.markdown:
```diff
@@ -27,3 +27,19 @@ The following devices are supported:
+
+***
+
+*\* Remote Access via Thread requires a compatible Thread Border Router.*
```
@copilot open a new pull request to apply changes based on [this feedback](https://github.com/home-assistant/home-assistant.io/pull/41829#discussion_r2535120950)

djansen1987 commented on source/_integrations/nuki_matter.markdown:
```diff
@@ -27,3 +27,19 @@ The following devices are supported:
+
+*\* Remote Access via Thread requires a compatible Thread Border Router.*
+
+[Nuki Documentation](https://help.nuki.io/hc/en-001/articles/14596875392017-Setting-up-your-Matter-integration)
```
@copilot open a new pull request to apply changes based on [this feedback](https://github.com/home-assistant/home-assistant.io/pull/41829#discussion_r2535120974)

djansen1987 commented on source/_integrations/nuki_matter.markdown:
```diff
@@ -27,3 +27,19 @@ The following devices are supported:
+
+*\* Remote Access via Thread requires a compatible Thread Border Router.*
+
+[Nuki Documentation](https://help.nuki.io/hc/en-001/articles/14596875392017-Setting-up-your-Matter-integration)
```
@copilot open a new pull request to apply changes based on [this feedback](https://github.com/home-assistant/home-assistant.io/pull/41829#discussion_r2535120995)

djansen1987 replied:
@copilot open a new pull request to apply changes based on [this feedback](https://github.com/home-assistant/home-assistant.io/pull/41829#r2535120995)

djansen1987 commented on source/_integrations/nuki_matter.markdown:
```diff
@@ -27,3 +27,19 @@ The following devices are supported:
 - [Nuki Smart Lock Pro](https://nuki.io/en-uk/products/smart-lock-pro-5th-gen)
 - [Nuki Smart Lock Ultra](https://nuki.io/en-uk/products/smart-lock-ultra)
+
+## Setting up your Matter integration requires Thread
```
@copilot open a new pull request to apply changes based on [this feedback](https://github.com/home-assistant/home-assistant.io/pull/41829#discussion_r2535121007)

djansen1987 commented on source/_integrations/nuki_matter.markdown:
```diff
@@ -27,3 +27,19 @@ The following devices are supported:
+
+## Setting up your Matter integration requires Thread
+
+To integrate your Smart Lock into your Smart Home, you need a suitable hub that supports **Matter via Thread**.
```
@copilot open a new pull request to apply changes based on [this feedback](https://github.com/home-assistant/home-assistant.io/pull/41829#discussion_r2535121025)

djansen1987 commented on source/_integrations/nuki_matter.markdown:
```diff
@@ -27,3 +27,19 @@ The following devices are supported:
+
+To integrate your Smart Lock into your Smart Home, you need a suitable hub that supports **Matter via Thread**.
+
+## 🏡 Home Assistant Connect ZBT-1 Support
```
@copilot open a new pull request to apply changes based on [this feedback](https://github.com/home-assistant/home-assistant.io/pull/41829#discussion_r2535121041)

djansen1987 commented on source/_integrations/nuki_matter.markdown:
```diff
@@ -27,3 +27,19 @@ The following devices are supported:
+
+| Type | Matter via Thread | Remote Access via Thread* |
+| :--- | :---: | :---: |
+| Home Assistant Connect ZBT-1 | ✅ | ✅ |
```
@copilot open a new pull request to apply changes based on [this feedback](https://github.com/home-assistant/home-assistant.io/pull/41829#discussion_r2535121054)

djansen1987 committed:
Revise Nuki Matter integration documentation

Updated the Matter integration documentation for Nuki devices, clarifying the requirement for a Thread border router and reorganizing supported devices section.

agners commented on source/_integrations/nuki_matter.markdown:
```diff
@@ -20,6 +20,17 @@ works_with:
+
+## Home Assistant Connect ZBT-1 support
+
+The Home Assistant Connect ZBT-1 supports Matter devices over Thread and can function as a Thread border router for remote access. For information on setting up the Connect ZBT-1 as a Thread border router, refer to the [Thread documentation](/integrations/thread/).
```
I'd remove the title of this section, and make this a second sentence of the above heading, e.g.:

You can turn your Home Assistant installation into a Thread border router e.g. by using the Home Assistant Connect ZBT-1. Refer to the [Turning Home Assistant into a Thread border router](/integrations/thread/#turning-home-assistant-into-a-thread-border-router) section of the Thread integration documentation

djansen1987 committed:
Update Matter integration setup instructions

Clarified the setup instructions for Matter integrations and removed redundant information about Home Assistant Connect ZBT-1.

agners commented on source/_integrations/nuki_matter.markdown:
```diff
@@ -20,6 +20,12 @@ works_with:
 
+## Setting up your Matter integrations requires Thread
+
+You can turn your Home Assistant installation into a Thread border router e.g. by using the Home Assistant [Connect ZBT-1](https://www.home-assistant.io/connectzbt1/). For more information on setting this up, go to [Turning Home Assistant into a Thread border router section](/integrations/thread/#turning-home-assistant-into-a-thread-border-router)
```
Now it makes it sound as if Home Assistant Thread border router is the only option, that was not my intention. I'd still leave the generic sentence there. E.g. like so:

```suggestion
## Setting up Nuki via Matter integration requires Thread

To use this integration, you need a Thread border router that supports Matter. For more information, refer to the [Thread documentation](/integrations/thread/). You can turn your Home Assistant installation into a Thread border router e.g. by using the Home Assistant [Connect ZBT-1](https://www.home-assistant.io/connectzbt1/). For more information on setting this up, go to [Turning Home Assistant into a Thread border router section](/integrations/thread/#turning-home-assistant-into-a-thread-border-router)
```

c0ffeeca7 committed:
Merge branch 'current' into patch-2

djansen1987 committed:
Update nuki_matter.markdown

djansen1987 committed:
Update nuki_matter.markdown

djansen1987 committed:
Update nuki_matter.markdown

added zbt-2

agners commented on source/_integrations/nuki_matter.markdown:
```diff
@@ -20,6 +20,12 @@ works_with:
 
+## Setting up your Nuki via Matter integrations requires Thread
+
+You can turn your Home Assistant installation into a Thread border router e.g. by using the Home Assistant [Connect ZBT-1](https://www.home-assistant.io/connectzbt1/) or [ZBT-2](https://www.home-assistant.io/connect/zbt-2/). For more information on setting this up, go to [Turning Home Assistant into a Thread border router section](/integrations/thread/#turning-home-assistant-into-a-thread-border- ...
```
```suggestion
To use this integration, you need a Thread border router that supports Matter. For more information, refer to the [Thread documentation](/integrations/thread/). You can turn your Home Assistant installation into a Thread border router e.g. by using the Home Assistant [Connect ZBT-1](https://www.home-assistant.io/connectzbt1/) or [ZBT-2](https://www.home-assistant.io/connect/zbt-2/). For more information on setting this up, go to [Turning Home Assistant into a Thread border router section](/integrations/thread/#turning-home-assistant-into-a-thread-border-router)
```

agners commented:
> Thanks, that button is not showing on the android app apparently, 

When you are logged in, it should appear on the android app too, you need to change to the "Files changed".

> so manual edit it. Hope it is good now, i guess we should also include the zbt-2 now it is released?

I agree, ZBT-2 should be mentioned too, thanks for adding that. Somehow the first sentence got lost again, but I'll readd it and approve the PR.

Thanks for the improvement!

agners committed:
Apply suggestions from code review

c0ffeeca7 reviewed (approved):
Thank you, @djansen1987 and @agners :+1:

djansen1987 commented:
Thanks for changing. I have looked again for that option but don't see it in the app. I only see this. Do I understand correctly that it is done now or still waiting for an approval.


![Screenshot_20251126_125928_GitHub.jpg](https://github.com/user-attachments/assets/f13577d5-3018-4dc7-b972-9d57e86d0079)

![Screenshot_20251126_125933_GitHub.jpg](https://github.com/user-attachments/assets/d0d4178a-ff21-4e8a-b23e-7041e1a66495)


> > Thanks, that button is not showing on the android app apparently, 
> 
> When you are logged in, it should appear on the android app too, you need to change to the "Files changed".
> 
> > so manual edit it. Hope it is good now, i guess we should also include the zbt-2 now it is released?
> 
> I agree, ZBT-2 should be mentioned too, thanks for adding that. Somehow the first sentence got lost again, but I'll readd it and approve the PR.
> 
> Thanks for the improvement!
